projects/igniteui-angular/src/lib/grids/grid/grid-filtering-ui.spec.ts

```diff
@@ -2146,24 +2146,23 @@ describe('IgxGrid - Filtering Row UI actions #grid', () => {
         }));
 
         it('Should close filter row when hide the current column', fakeAsync(() => {
-            pending('This issue is failing because of bug #');
             GridFunctions.clickFilterCellChip(fix, 'ProductName');
 
             // Check that the filterRow is opened
-            const filterUIRow = fix.debugElement.query(By.css(FILTER_UI_ROW));
+            let filterUIRow = fix.debugElement.query(By.css(FILTER_UI_ROW));
             expect(filterUIRow).not.toBeNull();
 
             // Add first chip.
             GridFunctions.typeValueInFilterRowInput('a', fix);
             tick(100);
 
             grid.getColumnByName('ProductName').hidden = true;
-            fix.detectChanges();
             tick(100);
+            fix.detectChanges();
 
             // Check that the filterRow is closed
-            expect(fix.debugElement.query(By.css(FILTER_UI_ROW))).toBeNull();
-            expect(grid.rowList.length).toBe(8);
+            filterUIRow = fix.debugElement.query(By.css(FILTER_UI_ROW));
+            expect(filterUIRow).toBeNull();
```

Comment:
You can add a check that the filtering is applied (the rowList.length is changed).